src/Components/NavigationLink/NavigationLink.js

```diff
@@ -0,0 +1,26 @@
+import { ListItem, ListItemText } from '@mui/material'
+import React from 'react'
+import PropTypes from 'prop-types';
+
+NavigationLink.propTypes = {
+    linkText: PropTypes.string.isRequired,
+}
+
+function NavigationLink({ linkText }) {
+    return (
+        <ListItem sx={{
+            padding: 0,
+        }}>
+            <ListItemText
+                primary={linkText}
+                fontWeight={'fontWeightMedium'}
+                primaryTypographyProps={{
+                    fontWeight: 'fontWeightMedium',
```

Comment:
Since you've already specified fontWeight: 'fontWeightMedium' in primaryTypographyProps, the fontWeight outside primaryTypographyProps is redundant and incorrect, you should remove it